src/main/java/net/dv8tion/jda/core/handle/GuildUpdateHandler.java

```diff
@@ -44,17 +44,16 @@ protected Long handleInternally(JSONObject content)
         GuildImpl guild = (GuildImpl) api.getGuildMap().get(id);
         Member owner = guild.getMembersMap().get(content.getLong("owner_id"));
         String name = content.getString("name");
-        String iconId = !content.isNull("icon") ? content.getString("icon") : null;
-        String splashId = !content.isNull("splash") ? content.getString("splash") : null;
+        String iconId = content.isNull("icon") ? null : content.getString("icon");
+        String splashId = content.isNull("splash") ? null : content.getString("splash");
         String region = content.getString("region");
         Guild.VerificationLevel verificationLevel = Guild.VerificationLevel.fromKey(content.getInt("verification_level"));
         Guild.NotificationLevel notificationLevel = Guild.NotificationLevel.fromKey(content.getInt("default_message_notifications"));
         Guild.MFALevel mfaLevel = Guild.MFALevel.fromKey(content.getInt("mfa_level"));
         Guild.ExplicitContentLevel explicitContentLevel = Guild.ExplicitContentLevel.fromKey(content.getInt("explicit_content_filter"));
         Guild.Timeout afkTimeout = Guild.Timeout.fromKey(content.getInt("afk_timeout"));
-        VoiceChannel afkChannel = !content.isNull("afk_channel_id")
-                ? guild.getVoiceChannelsMap().get(content.getLong("afk_channel_id"))
-                : null;
+        VoiceChannel afkChannel = content.isNull("afk_channel_id")
+                ? null : guild.getVoiceChannelsMap().get(content.getLong("afk_channel_id"));
         TextChannel systemChannel = !content.isNull("system_channel_id")
```

Comment:
As you've already removed the `!` in the three lines above could you also change it here?